hudi-client/src/test/java/org/apache/hudi/common/HoodieTestDataGenerator.java

```diff
@@ -136,6 +151,9 @@ public static GenericRecord generateGenericRecord(String rowKey, String riderNam
     rec.put("end_lat", rand.nextDouble());
     rec.put("end_lon", rand.nextDouble());
     rec.put("fare", rand.nextDouble() * 100);
+    if (isDeleteRecord) {
```

Comment:
can we test with all other fields null, in case of deletes.. (ofc except _row_key and what you need to generate partitionpath)